### PR TITLE
Several small fixes and default to using NFS for performance

### PR DESCRIPTION
**Pull request by seajoshc**
-Adds .vagrant/ to .gitignore and removes from origin
-Update README
-Add NFS for performance
-Increase default VM size and add option for dynamic sizing based on Host resources
-rvm gpg key import now forces port 80 to get around potentially restrictive firewalls

### CODE REVIEW

**Comment by seajoshc** [comment]
@johncraig2 wanna CR me? :)

File: README.md [hunk 1]
```diff
@@ -1,25 +1,60 @@
...
-3. Install Ansible - https://devopsu.com/guides/ansible-mac-osx.html or `brew install ansible` if you use Homebrew (http://brew.sh/)
-3. Clone this repo
-4. Go into the repo
-5. To start your new dev environment run 'vagrant up'.  This will take several minutes to create and provision the virtual machine.
-6. To ssh to your new dev environment run 'vagrant ssh'
+# Getting Started
+1. Install Virtualbox - https://www.virtualbox.org/wiki/Downloads
+2. Install Vagrant - https://www.vagrantup.com/downloads.html
+3. Install Ansible - `brew install ansible` or https://devopsu.com/guides/ansible-mac-osx.html
+3. Clone this repo with 'git clone git@github.groupondev.com:SRE/sre_dev.git'
```
Comment (johncraig2) [inline README.md]: I think you'll want to revise this url. :-)

**Comment by seajoshc** [comment]
Hahaha can you tell I copy/pasted from another private repo? ;)  

Nice catch!

**Comment by johncraig2** [comment]
LGTM! Ship it! :-)

**Comment by seajoshc** [comment]
Danke!

**Comment by johncraig2** [comment]
awesome changes - very useful - thanks so much!

**Pull request merged**